src/ctypes/complexL.ml

```diff
@@ -40,10 +40,4 @@ external arg : t -> LDouble.t = "ctypes_ldouble_complex_cargl"
 external exp : t -> t = "ctypes_ldouble_complex_cexpl"
 external log : t -> t = "ctypes_ldouble_complex_clogl"
 external pow : t -> t -> t = "ctypes_ldouble_complex_cpowl"
-
-let inv x =
-  if Sys.win32 then
-    (* complex division not supported by MSVC C runtime *)
-    pow x negone
-  else
-    div one x
+external inv : t -> t = "ctypes_ldouble_complex_cinvl"
```

Comment:
Seems to be missing. When you write it, also include a small test so that the function is not overlooked again.

Reply:
It was implemented (`CCC_PRIM_INV_OP` in src/ctypes/ctypes_complex_compatibility.h) but still better to add tests. PR updated.